tests/dragonfly/generic_test.py

```diff
@@ -149,7 +149,7 @@ async def test_reply_guard_oom(df_factory, df_seeder_factory):
 @pytest.mark.asyncio
 async def test_denyoom_commands(df_factory):
     df_server = df_factory.create(
-        proactor_threads=1, maxmemory="256mb", oom_deny_commands="get", oom_deny_ratio=0.7
+        proactor_threads=1, maxmemory="256mb", oom_deny_commands="get", rss_oom_deny_ratio=0.7
```

Comment:
you should not replace oom_deny_ratio with  rss_oom_deny_ratio in this test
This test checks that we return oom for denyoom commands
so now we dont have the ration anymore we will return oom when we reach maxmemory set by user

Reply:
Yes, will fix